docs/static/pipeline-configuration.asciidoc

```diff
@@ -46,21 +52,23 @@ filter {
 output {
   ...
 }
-----------------------------------
+-----
+
+Each section contains configuration options for one or more plugins. 
+If you specify multiple filters, they are applied in the order they appear in the configuration file.
 
-Each section contains the configuration options for one or more plugins. If you specify
-multiple filters, they are applied in the order of their appearance in the configuration file.
+TIP: When you are ready to deploy a pipeline beyond your local machine, add the pipeline config file to <<logstash-settings-file,`logstash.yml`>> using the `pipeline.id` setting. 
```

Comment:
Placeholder: Add a note about next steps so that we don't abandon users are command line/demo phase.

Reply:
A good launching point for `pipelines.yml`?